### PR TITLE
Add properties search

### PR DESCRIPTION
**Pull request by benbaumann95**
Description of change
Probably best to go through commits..
But in summary:

Adds relevant frontend libraries
Adds axios to talk to our API
Adds jest unit testing
Add necessary components to render Properties table and view pages
Adds very basic error handling
Add PropType validations
Add Router (https://nextjs.org/docs/api-reference/next/router) so we can have pages and routing

### CODE REVIEW

File: styles/all.scss [hunk 1]
```diff
@@ -0,0 +1 @@
+@import 'node_modules/govuk-frontend/govuk/all';
```
Comment (EGiataganas) [inline styles/all.scss]: 👍

File: utils/api/properties.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import axios from 'axios'
+
+console.log(process.env.ENDPOINT_API)
```
Comment (NeilvB) [inline utils/api/properties.js]: Can we ✂️ this?

**Review by NeilvB — APPROVED** [review]
Just a `console.log` I think we can remove

File: utils/api/properties.js (hunk 2 above)
Comment (benbaumann95) [reply utils/api/properties.js]: Ah might have been left in an older commit. Should be removed in the most up to date commit but thanks for flagging

Comment (NeilvB) [reply utils/api/properties.js]: Ah of course, yep it is. Thanks

File: utils/api/properties.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  return data
+}
+
+export const getProperty = async (propertyReference) => {
+  const { data } = await axios.get(
+    `${NEXT_PUBLIC_PROPERTIES_ENDPOINT_API}/${propertyReference}`
+  )
+
+  return data
+}
```
Comment (EGiataganas) [inline utils/api/properties.js:19]: Is there any particular reason for deleting the `/properties` and required it as part of the environment variable? Isn't better to have only one env var which will be the `API_URL`?

**Review by EGiataganas — APPROVED** [review]

**Pull request merged**